defaults/main.yml

```diff
@@ -31,3 +33,17 @@ couchdb_pki_cert_suffix: '.pem'
 
 couchdb_init: systemd
 couchdb_activate: True
+couchdb_async_io_threads: 16
```

Comment:
https://github.com/apache/couchdb/blob/master/rel/overlay/etc/vm.args

This is what is included with the default install. I've kept the default here as the initial value. the vm.args file is always present in couch 2.0 installs from what I can tell